README.md

```diff
@@ -6,11 +6,11 @@ This repository contains the [System.Device.Gpio](https://www.nuget.org/packages
 
 The repository also contains [IoT.Device.Bindings](https://www.nuget.org/packages/Iot.Device.Bindings), a growing set of community-maintained [device bindings](src/devices/README.md) for IoT components.
 
-Note: System.Device.Gpio is in early preview. It is not yet supported and will continue to change. It is currently published to myget (requires use of [nuget.config](samples/led-blink/nuget.config)).
+**Note**: System.Device.Gpio is in early preview. It is not yet supported and will continue to change. It is currently published to myget (requires use of [nuget.config](samples/led-blink/nuget.config)).
 
-## How to Install and Get Started
+## How to Install
 
-You can install the latest daily pre-release build of the .NET Core System.Device.Gpio and IoT.Device.Bindings nuget packages from the [dotnet.myget.org feed](https://dotnet.myget.org/feed/dotnet-core/package/nuget/System.Device.Gpio).
+You can install the latest daily pre-release build of the .NET Core System.Device.Gpio and IoT.Device.Bindings NuGet packages from the [dotnet.myget.org feed](https://dotnet.myget.org/feed/dotnet-core/package/nuget/System.Device.Gpio).
 
 #### NuGet.exe
```

Comment:
IMO this whole section and similar should go elsewhere (`how to install` perhaps)

Reply:
The plan is to update that area later.  This was more focused on moving the roadmap content.

Reply:
sounds good